src/resources/nfts/simplehash/utils.ts

```diff
@@ -90,81 +91,44 @@ export function getNetworkFromSimpleHashChain(chain: SimpleHashChain): Network {
   }
 }
 
-/**
- * Filters out NFTs that do not have a name, collection name,
- * contract address, or token id, Gnosis NFTs that are not POAPs,
- * NFTs that are not on a supported `Network`, and Polygon NFTs
- * that are not present in the provided allowlist.
- * @param nfts array of `SimpleHashNFT`s
- * @param polygonAllowlist array of Polygon nft contract addresses to allowlist
- * @returns filtered array of `ValidatedSimpleHashNFT`s
- */
-export function filterSimpleHashNFTs(nfts: SimpleHashNFT[], polygonAllowlist?: PolygonAllowlist): ValidatedSimpleHashNFT[] {
-  return nfts
-    .filter(nft => {
-      const lowercasedContractAddress = nft.contract_address?.toLowerCase();
-      const network = getNetworkFromSimpleHashChain(nft.chain);
-
-      const isMissingRequiredFields = !nft.name || !nft.collection?.name || !nft.contract_address || !nft.token_id || !network;
-      const isPolygonAndNotAllowed =
-        polygonAllowlist && nft.chain === SimpleHashChain.Polygon && !polygonAllowlist[lowercasedContractAddress];
-      const isGnosisAndNotPOAP = nft.chain === SimpleHashChain.Gnosis && lowercasedContractAddress !== POAP_NFT_ADDRESS;
-
-      if (isMissingRequiredFields || isPolygonAndNotAllowed || isGnosisAndNotPOAP) {
-        return false;
-      }
-
-      return true;
-    })
-    .map(nft => ({
-      ...nft,
-      name: nft.name!,
-      contract_address: nft.contract_address,
-      chain: getNetworkFromSimpleHashChain(nft.chain),
-      collection: { ...nft.collection, name: nft.collection.name! },
-      token_id: nft.token_id!,
-    }));
-}
-
 /**
  * Maps a `SimpleHashNFT` to a `UniqueAsset`.
  * @param nft `SimpleHashNFT`
  * @returns `UniqueAsset`
  */
-export function simpleHashNFTToUniqueAsset(nft: ValidatedSimpleHashNFT, address: string): UniqueAsset {
+export function simpleHashNFTToUniqueAsset(nft: SimpleHashNft, address: string): UniqueAsset {
```

Comment:
loosened the simplehash nft type returned by the new `nfts` graphql query, so had to make some adjustments throughout this function to accommodate for that

Reply:
the purpose of loosening simplehash types is to prevent any graphql error responses caused by simplehash not respecting their own types